src/graph/graph.c

```diff
@@ -215,9 +215,6 @@ Graph *Graph_New(size_t node_cap, size_t edge_cap) {
     // Force GraphBLAS updates and resize matrices to node count by default
     Graph_SetMatrixPolicy(g, SYNC_AND_MINIMIZE_SPACE);
 
-    // Graph_New can only be invoked from writing contexts
-    Graph_AcquireWriteLock(g);
```

Comment:
I thought we had deleted this? I might have re-introduced it by accident, I'm not sure.